backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.data.Match;
+import com.google.sps.data.Participant;
+import com.google.sps.FindMatchQuery;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+* Servlet that returns some example content.
+*/
+@WebServlet("/add-participant")
+public class AddParticipantServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Request parameter values
+    // TODO: Error check input values (before allowing to submit)
+    UserService userService = UserServiceFactory.getUserService();
+    String email = userService.getCurrentUser().getEmail();
+    String ldap = email.split("@")[0];
```

Comment:
Is there a reason the 0 is in brackets as well as not within thee. split call, split takes two arguments the reg-ex and the amount you wish to split by. Since we know this is an email and there can only be 1 @ symbol I think you can do **_email.split("a",1)_**

Reply:
The second argument is optional, so I just omitted it.

Reply:
Also, there's a [0] because I'm taking the first element of the array (ldap). Example: hello@gmail.com is split into ["hello", "gmail.com"], and I just want the "hello" part.